react/Overlay/index.jsx

```diff
@@ -54,13 +47,14 @@ class Overlay extends Component {
   render() {
     const { children, className } = this.props
     const domProps = omit(this.props, nonDOMProps)
+    const Wrapper = bodyTallerThanWindow() ? React.Fragment : RemoveScroll
```

Comment:
Why do we have to check this ? Shouldn't we render RemoveScroll in all cases ? If it's necessary, could you add a comment why ?

Reply:
Done and issue created too 👍 https://github.com/cozy/cozy-ui/issues/1599